src/main/java/org/folio/bulkops/domain/converter/HoldingsNoteListConverter.java

```diff
@@ -34,15 +46,27 @@ public List<HoldingsNote> convertToObject(String value) {
 
   @Override
   public String convertToString(List<HoldingsNote> object) {
-    return object.stream()
+    var str = object.stream()
       .filter(Objects::nonNull)
-      .map(note -> String.join(ARRAY_DELIMITER,
-        escape(HoldingsReferenceHelper.service().getNoteTypeNameById(note.getHoldingsNoteTypeId())),
-        escape(note.getNote()),
-        booleanToStringNullSafe(note.getStaffOnly()),
-        note.getTenantId(),
-        note.getHoldingsNoteTypeId()))
+      .map(note -> {
+        var noteTypeName = note.getHoldingsNoteTypeName();
+        if (isNull(noteTypeName)) {
+          try {
+            noteTypeName = HoldingsReferenceHelper.service().getNoteTypeNameById(note.getHoldingsNoteTypeId());
+          } catch (NotFoundException e) {
+            log.error("Holding note type with id = {} not found : {}", note.getHoldingsNoteTypeId(), e.getMessage());
+            noteTypeName = "";
```

Comment:
String noteType name can be initialised as empty string before try-catch block.

Reply:
Fixed.